voicevox/speaker_info.py

```diff
@@ -0,0 +1,59 @@
+# voicevox - speaker_info
+
+from typing import List
+
+from .types.speaker_info import SpeakerInfoType, StyleInfoType
+
+class StyleInfo:
+    """Return style info
+
+    Attributes
+    ----------
+    id: int
+        style id
+    icon: str
+        base64 encoded icon
+    portrait: str
+        base64 encoded portrait image
+    voice_samples: list[str]
+        raw voice_data ???
+    """
+
+    def __init__(self, payload: StyleInfoType):
+        self.__data = payload
+
+    @property
+    def id(self) -> int:
+        return self.__data["id"]
+    
+    @property
+    def icon(self) -> str:
+        return self.__data["icon"]
+    
+    @property
+    def portrait(self) -> str:
+        return self.__data["portrait"]
+    
+    @property
+    def voice_samples(self) -> list[str]:
+        return self.__data["voice_samples"]
+    
```

Comment:
Please add one more new line.

Reply:
I am sorry for my late responses, I hope this time it is done right.
Also I remember checking a box allowing revisers to edit the code, if I respond late you may want to do that because I will be away for some time soon.